marinerescue-frontend/src/pages/HomePage.scss

```
@@ -0,0 +1,237 @@
+@import '../styles/breaks';
+@import '../styles/colors';
+
+*{
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+}
+
+// navbar styling source: https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_topnav_equal_width
+
+.navbar {
+    display: flex;
+    align-items:center;
+    width: 100%;
+    background-color: $blue;
+    overflow: auto;
+    padding: 14px 50px;
+
+    #nav-bar-button {
+        background-color: $sand;
+        border: none;
+        color: $grape;
+        width: 180px;
+        padding: 20px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 20px;
+        margin: 30px 2px;
+        border-radius: 30px;
+        font-family : inherit;
+    }
+}
+  
+.navbar a {
+    float: left;
+    padding: 12px;
+    color: white;
+    text-decoration: none;
+    font-size: 20px;
+    width: 25%; 
+    text-align: center;
+}
+  
+.navbar a:hover {
+    color: $dark;
+}
+
+.navbar-logo{ 
+    margin-bottom: -5px;
+
+    img{
+       width: 100%;
+       height: auto; 
+    }
+}
+  
+@media screen and (max-width: 500px) {
+    .navbar a {
+      float: none;
+      display: block;
+      width: 100%;
+      text-align: left;
+    }
+}
+
+// button styling source: https://www.w3schools.com/howto/howto_css_round_buttons.asp
+
+button {
+    background-color: $blue;
+    border: none;
+    color: white;
+    width: 180px;
+    padding: 20px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 20px;
+    margin: 30px 2px;
+    border-radius: 30px;
+    font-family : inherit;
+
+}
+
+.row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 40px;
+}
```

Improve responsivity
```suggestion
    .row {
        display: flex;
        flex-direction: row;
        align-items: center;
        padding: 10px;
        @media screen and (max-width: $break-small) {
                 flex-direction: column;
         } 
    }
```